examples/oidc/thunderbird/OAuth2Providers.jsm

```diff
@@ -0,0 +1,195 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
Mmmh this could be an issue.

Things in an Apache project shall be licensed as Apache V2, this could (should?) trigger a legal discussion...

Reply:
https://issues.apache.org/jira/browse/LEGAL-598 here we go

Reply:
> Source code under MPL can't be included in an ASF release as it belongs to Category B. [1]

Maybe we should capture a screenshot instead of a file code?

Reply:
Well IMO it falls down in the cheating categorie. Including only the diff in the README would potentially be preferable and is definitly acceptable.

https://svn.apache.org/repos/asf/infrastructure/site/trunk/archive/legal/3party.mdtext -> this falls in the reciprocal categorie. This is an exemple so we "could" be in a corner case.

Reply:
`Source code under MPL can't be included in an ASF release as it belongs to Category B. [1]`

We got our answer quickly. Please only state in the README how to modify that file ;-)